File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        int count=0;
        for (int i=0;i<nums.length;i++){
            for(int j=i+1;j<nums.length;j++){
                if(nums[i]==nums[j]){
                    count++;
                }
              
            }
        }
        if(count==0){
            return false;
        }
        return true;
        
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution(object):
    def topKFrequent(self, nums, k):
        freq = {}  # frequency dictionary

        # Count frequency of each number
        for num in nums:
            if num in freq:
                freq[num] += 1
            else:
                freq[num] = 1

        # Sort the items by frequency in descending order
        sorted_items = sorted(freq.items(), key=lambda x: x[1], reverse=True)

        # Extract the top k elements
        result = [item[0] for item in sorted_items[:k]]

        return result
